lib/saluki-components/src/destinations/datadog/metrics/request_builder.rs

```diff
@@ -343,7 +343,7 @@ where
     O: ObjectPool<Item = BytesBuffer> + 'static,
 {
     let write_buffer = buffer_pool.acquire().await;
-    Compressor::from_scheme(CompressionScheme::zlib_default(), write_buffer)
+    Compressor::from_scheme(CompressionScheme::zstd_default(), write_buffer)
```

Comment:
Should [this header value](https://github.com/DataDog/saluki/blob/0d57fd8ae3ac44b02e9c99712584976a55ed62d3/lib/saluki-components/src/destinations/datadog/metrics/request_builder.rs#L20) be updated to `zstd`?